conda/recipes/cugraph/meta.yaml

```diff
@@ -19,34 +19,36 @@ build:
   number: {{ GIT_DESCRIBE_NUMBER }}
   string: cuda{{ cuda_major }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
   script_env:
-    - CC
-    - CXX
-    - CUDAHOSTCXX
     - PARALLEL_LEVEL
+  ignore_run_exports_from:
+    - nvcc_linux-64 # [linux64]
+    - nvcc_linux-aarch64 # [aarch64]
 
 requirements:
   build:
-    - python x.x
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }} {{ cuda_version }}
+    - sysroot_linux-64 {{ sysroot_version }}  # [linux64]
+    - sysroot_linux-aarch64 {{ sysroot_version }}  # [aarch64]
+  host:
+    - python
     - cython>=0.29,<0.30
-    - libcugraph={{ version }}
-    - libraft-headers {{ minor_version }}
-    - pyraft {{ minor_version }}
-    - cudf={{ minor_version }}
+    - libcugraph={{ version }}.*
+    - libraft-headers {{ minor_version }}.*
+    - pyraft {{ minor_version }}.*
+    - cudf={{ minor_version }}.*
     - ucx-py {{ ucx_py_version }}
     - ucx-proc=*=gpu
     - cudatoolkit {{ cuda_version }}.*
-    - libraft-headers {{ minor_version }}
+    - libraft-headers {{ minor_version }}.*
   run:
-    - python x.x
+    - python
     - pylibcugraph={{ version }}
     - libcugraph={{ version }}
     - libraft-headers {{ minor_version }}
     - pyraft {{ minor_version }}
     - cudf={{ minor_version }}
-    - dask-cudf {{ minor_version }}
-    - dask-cuda {{ minor_version }}
-    - dask>=2022.03.0
-    - distributed>=2022.03.0
```

Comment:
cugraph still directly depends on all of these packages and I prefer to avoid assuming that they'll all be properly installed transitively.  Is there a reason they're being a removed here?  If so, can it be documented somewhere in this file that we still use these packages?

Reply:
Thank you for catching this Rick, this was not expected at all. PR updated to fix this